posthog/api/survey.py

```diff
@@ -679,6 +679,8 @@ def _create_or_update_targeting_flag(
 class SurveyViewSet(TeamAndOrgViewSetMixin, viewsets.ModelViewSet):
     scope_object = "survey"
     queryset = Survey.objects.select_related("linked_flag", "targeting_flag", "internal_targeting_flag").all()
+    filter_backends = [filters.SearchFilter]
+    search_fields = ["name", "description"]
```

Comment:
how do we know if this is case sensitive/insensitive?

Reply:
`SearchFilter` uses case-insensitive `icontains` lookup by default [source](https://www.django-rest-framework.org/api-guide/filtering/)